src/public/modules/forms/base/css/end-page.css

```diff
@@ -6,3 +6,11 @@
 #end-page-header #end-page-timestamp {
   font-weight: bold;
 }
+
+#end-page-header #end-page-responseid {
+  font-size: 12px;
+}
+
+#end-page-header #end-page-responseid #end-page-id {
+  text-transform: lowercase;
+}
```

Comment:
correct me if i'm wrong but wouldn't this also apply styling to `end-page-header` where there wasn't previously?

Reply:
If I'm not wrong, the style is applied to `end-page-responseid` which is a descendant of the `end-page-header` element. It only applies to `end-page-header` as well if I insert a comma between the two, which will then make them two separate selectors.

Reply:
ah ok you're right i think how it works is through the [descendant combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator)!